SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -320,12 +322,61 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
 
   bool incompressible;
   bool sustaining_terms;
+  bool axisymmetric;
 
   /*!
    * \brief A virtual member. Get strain magnitude based on perturbed reynolds stress matrix
    * \param[in] turb_ke: turbulent kinetic energy of the node
    */
   void SetPerturbedStrainMag(su2double turb_ke);
+  
+  /*!
+   * \brief Add contribution due to axisymmetric formulation to 2D residual
+   */
+  inline void ResidualAxisymmetric(su2double alfa_blended, su2double zeta){
+
+    if (Coord_i[1] < EPS) {
+      return;
+    }
+    
+    else{
+      
```

Comment:
The idea of the early return was not to need the else xD

Reply:
Oupsi haha